### PR TITLE
Documentation fixes 

### PR DESCRIPTION
**Pull request by Jimbo4350**

### CODE REVIEW

File: doc/getting-started/install.md [hunk 1]
```diff
@@ -50,23 +50,23 @@ If you are using a different flavor of Linux, you will need to use the correct p
     mkdir -p ~/.local/bin
     mv cabal ~/.local/bin/
 
-Verify that ~/.local/bin is in your PATH
+Verify that ~/.local/bin is in your PATH:
```
Comment (newhoggy) [inline doc/getting-started/install.md:53]: Add backticks to `~/.local/bin` above and to `PATH` above and below.

File: doc/getting-started/install.md [hunk 2]
```diff
@@ -50,23 +50,23 @@ If you are using a different flavor of Linux, you will need to use the correct p
...
 
     source .bashrc
 
-Update cabal
+Update cabal:
 
     cabal update
 
-Confirm that you installed cabal version `3.2.0.0`.
+Confirm that you installed cabal version `3.2.0.0`:
```
Comment (newhoggy) [inline doc/getting-started/install.md:69]: I reckon we can use `3.4.0.0` now since it has been released and we're using it in GH.

**Review by newhoggy — APPROVED** [review]

**Review by olgahryniuk — APPROVED** [review]

**Pull request merged**